stringutil/stringutil.go

```diff
@@ -7,6 +7,19 @@ import (
 	"unicode"
 )
 
+// Truncate returns the "n" left characters of the string.
+func Truncate(s string, n int) string {
+	if n <= 0 {
+		return ""
+	}
+
+	if len([]rune(s)) <= n {
```

Comment:
[RuneCountInString](https://pkg.go.dev/unicode/utf8#RuneCountInString) is faster than doing the slice conversion.

```suggestion
	if utf8.RuneCountInString(s) <= n {
```